sdk/src/client/node_api/core/routes.rs

```diff
@@ -103,6 +103,23 @@ impl ClientInner {
             .await
     }
 
+    /// Returns the totally available Mana rewards of an account or delegation output decayed up to epochEnd index
```

Comment:
```suggestion
    /// Returns the total available Mana rewards of an account or delegation output decayed up to epochEnd index
```

Reply:
Also what is epochEnd?

Reply:
It's the epoch until which the Mana rewards are calculated